packaging/tags.py

```diff
@@ -733,9 +732,96 @@ def _manylinux_tags(linux, arch):
                     yield linux.replace("linux", legacy_tag)
 
 
+def _get_host_platform():
```

Comment:
CI failures seem to just be for lack of lack of test coverage in packaging/tags.py due to using [`python -m coverage report -m --fail-under 100`](https://travis-ci.com/github/pypa/packaging/jobs/430438831#L666)

That seems hard to address by actually adding coverage; at lot of `_get_host_platform` will be unused on any given platform, so no single platform is going to reach 100% coverage. And even if you could sum the coverage reports together somehow, there would still uncovered code for a lot of niche platforms aren't in the CI. It was no different before, just outside the view of the coverage report (since it was in distutils).

I'm by no means an expert on pip/packaging/etc (just interested in a fix ever since filing https://bugs.python.org/issue38989). But this seems to me like a legit use for [`# pragma: no cover`](https://coverage.readthedocs.io/en/coverage-5.3/excluding.html#excluding), and tags.py aleady seems to already use that in many similar cases

```suggestion
def _get_host_platform():  # pragma: no cover
```

Reply:
I agree with your assessment and solution, but one of the maintainers need to confirm this 🙂